Cartfile.private

```diff
@@ -1,6 +1,5 @@
 github "mapbox/MapboxGeocoder.swift" ~> 0.10
-github "Quick/Quick" ~> 2.0
-github "Quick/Nimble" ~> 8.0
-github "CedarBDD/Cedar" ~> 1.0
```

Comment:
#3034 restores the Cedar dependency to avoid a compiler error, though it does so using the mapbox-events-ios checkout instead of directly depending on the framework via Carthage.